src/filterRows.js

```diff
@@ -189,7 +189,7 @@ function guessFilter(keyword = '') {
         }
     }
 
-    if (keyword.split(':').length === 2) {
+    if (keyword.split(':').length === 2 && keyword.split(':').every(v => isNumber(v.trim()))) {
```

Comment:
And in the case of a date filter, what does it look like?

Example: 2024/01/05:2024/01/10

It can be changed like this (?):
```
function isValidDate(dateString) {
  var date = new Date(dateString);
  return date instanceof Date && !isNaN(date);
}

if (keyword.split(':').length === 2 && keyword.split(':').every(v => isNumber(v.trim()) || isValidDate(v.trim()) )) {
```